handler.go

```diff
@@ -63,7 +68,7 @@ func (m *methods) findMethod(method string) (*caller, bool) {
 	return f, ok
 }
 
-func (m *methods) callLoopEvent(c *Channel, event string) {
+func (m *methods) callLoopEvent(c *Channel, event string, errors ...error) {
```

Comment:
Variadic function parameters to cascade whatever `closeChannel` was receiving in its variadic params